roles/keycloak/tasks/main.yml

```diff
@@ -43,7 +55,7 @@
     host: "%"
     priv: "{{ keycloak_database_name }}.*:ALL"
 
-- name: Disable pxc strict mode
+- name: Enable pxc strict mode
```

Comment:
I think Disable is correct

Reply:
https://github.com/vexxhost/atmosphere/blob/main/roles/keycloak/tasks/main.yml#L78
And this should be `Enable` i think